eng/common/pipelines/templates/steps/git-push-changes.yml

```diff
@@ -0,0 +1,54 @@
+parameters:
+  PRBranchName: not-specified
+  PROwner: azure-sdk
+  CommitMsg: not-specified
+  RepoOwner: Azure
+  RepoName: $(Build.Repository.Name)
+  PushArgs:
+  WorkingDirectory: $(System.DefaultWorkingDirectory)'
+  ScriptDirectory: eng/common/scripts
+  SkipCheckingForChanges: false
+
+steps:
```

Comment:
Just to close the conversation I agree with splitting this into its own template.